### ISSUE TITLE
[MU4 Issue] Image capture tool not implemented

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#15912]
**Describe the bug**
The image capture tool from MU3 is not implemented

This is known, of course,e but it is perceived to be helpful to have an issue here on GitHub to track it.

**Comment by barin71** [comment]
Да, эта функция нужна для быстрого захвата фрагмента и сохранения его.

**Comment by zorba77** [comment]
Any chance to have back this feature in 4.3?

**Comment by scorster** [comment]
Lots of avid .org rumblings and requests here:

https://musescore.org/en/node/341932

**Comment by cbjeukendrup** [comment]
> Given that "_This comment was marked as duplicate_" can someone please provide a link to the original or official issue?

This _is_ the official issue. That "duplicate" sentence did not relate to this issue as a whole, but to two comments that were just saying that this would be a useful feature, which is not informative so doesn't need to be seen. (Understandable misunderstanding, though, if you're not completely familiar with GitHub's UI.)

**Comment by scorster** [comment]
_This is the official issue._

Thank you for letting us know! Now I understand about hiding non informative messages.

_Understandable misunderstanding_

That's hilarious. And thanks for understanding!

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/378406

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/388293

**Comment by depth42** [comment]
I am a music school teacher and I really need the quick and high quality image capture to copy excerpts into my MS Word documents for my work sheets.
The macOS screenshot feature is not a full replacement as it is pixel based, always copies an opaque background and is harder to get precise.
The feature in MS3 was perfect for me.

**Comment by josiasmat** [comment]
I am also eagerly awaiting the implementation of this feature. In the meantime, the workaround I use is:
- export as SVG;
- open the SVG in Inkscape and perform the desired cropping;
- import the SVG file into the final document.
In Word/Libreoffice, the SVG can also be cropped directly.
